src/app/actions/anual-budget/action.ts

```diff
@@ -113,8 +113,8 @@ const generateAnualBudgetTeamMembersItems = (
             teamMemberId: item.teamMemberId!,
             memberRole: item.role,
             //If the team member has assigned "custom" workingMonths, those months will be used to calculate the amount of hours in total.
-            hours: item.workingMonths && item.workingMonths > 0 ? item.hours * item.workingMonths * 4 : item.hours * duration,
-            remainingHours: item.hours * duration,
+            hours: Math.ceil(item.workingMonths && item.workingMonths > 0 ? item.hours * item.workingMonths * 4.33333 : item.hours * duration),
```

Comment:
Extraería el 4.333 a una constante. Si mal no recuerdo hay un archivo de constantes. 

Por ejemplo: `const WEEKS_IN_A_MONTH = 51/12`

```suggestion
            hours: Math.ceil(item.workingMonths && item.workingMonths > 0 ? item.hours * item.workingMonths * WEEKS_IN_A_MONTH : item.hours * duration),
```

El resto all good 👌